src/molecules/formfields/SelectField/index.js

```diff
@@ -4,15 +4,24 @@ import classnames from 'classnames';
 import Tooltip from 'atoms/Tooltip';
 
 import { renderOption, renderPlaceholder } from 'utils/fieldUtils';
-
-import styles from 'molecules/formfields/shared/formfields.module.scss';
+import styles from './select_field.module.scss';
 
 function renderSelectOptions(options) {
   if (!options) return null;
 
   return options.map(renderOption);
 }
 
+const renderToolTip = (message) => {
+  if (!message) return null;
```

Comment:
You brought up a really good point. It won't matter for this particular case but I did some research and it shouldn't be an issue if you're using `React.Children.map` or if you `toArray` on the children before mapping.